features/step_definitions/common_steps.rb

```diff
@@ -52,16 +52,23 @@ def group_common(index)
 end
 
 Then(/^I should see '([^\"]*)' header$/) do |header|
-  page_header = common_page.content.page_header.text
-  expect(page_header).to have_content header
+  expect(common_page.content.h1.text).to have_content header
+end
+
+Then(/^I should see header '([^\"]*)'$/) do |header|
+  expect(common_page.content.h2.text).to have_content header
 end
 
 Then(/^I am taken to the Help with Fees homepage$/) do
   expect(current_path).to eq '/'
 end
 
 Then(/^I should see step (\d+) of 20$/) do |step_number|
-  expect(common_page.content.p.text).to have_content 'Step ' + step_number + ' of 20'
+  expect(common_page.content.p.text).to eq 'Step ' + step_number + ' of 20'
+end
+
+Then(/^I see step (\d+) of 20$/) do |step_number|
+  expect(common_page.heading_secondary.text).to eq 'Step ' + step_number + ' of 20'
```

Comment:
Same suggestion with above about string interpolation.

i.e.  "Step #{step_number} of 20"